stock_prices/stock_prices.py

```diff
@@ -3,7 +3,15 @@
 import argparse
 
 def find_max_profit(prices):
-  pass
+  maxProfit = 0
+  for index, price in enumerate(prices):
+    ##for all indexes after the current index, if the 2nd index value is larger, get the difference between the two
+      for index, nextPrice in enumerate(prices):
+        if nextPrice > price:
```

Comment:
There is a test that has an answer of -10 as the max profit, so you'll need to be checking all of the later entries minus the earlier one that you're on in the loop.